components/admin_console/billing/company_info_edit.scss

```diff
@@ -5,6 +5,7 @@
     background-color: var(--sys-center-channel-bg);
     border-radius: 4px;
     box-shadow: 0 2px 3px rgba(0, 0, 0, 0.08);
+    isolation: isolate;
```

Comment:
Why the fix is one property? As I say on the summary ideally should be at level component but I'm having some issue following the z-index between inputs that would make the fix much bigger. This property is a hidden gem of CSS, and it isn't new, that help us to make it reasonable to manage z-index values. This property does one thing and it is create a new stacking context, more info on this [post from Josh w Comeau](https://www.joshwcomeau.com/css/stacking-contexts/#airtight-abstractions-with-isolation).

Reply:
Awesome post! Didn't know about `isolation`, either. I can vouch for the `z-context` plugin's awesomeness, but haven't tried the other.